diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocFormatting.kt

```diff
@@ -200,10 +200,14 @@ class KdocFormatting(configRules: List<RulesConfig>) : DiktatRule("kdoc-formatti
                     .filter { basicTagsOrdered.contains(it.knownTag) }
                     .map { it.node }
 
-                basicTagsOrdered.forEachIndexed { index, tag ->
-                    val tagNode = kdocTags.find { it.knownTag == tag }!!.node
-                    kdocSection.addChild(tagNode.clone() as CompositeElement, basicTagChildren[index])
-                    kdocSection.removeChild(basicTagChildren[index])
+                val correctKdocOrder = basicTags
+                    .toMutableList()
```

Comment:
Is it needed here? If you are assigning sorted list to a new variable, you don't need in-place sorting and immutable list should be fine.

Reply:
Fixed